### PR TITLE
Add spark flash loan callback and gnosis chain deployment scripts

### PR DESCRIPTION
**Pull request by zodahu**

### CODE REVIEW

**Review by hihiben — APPROVED** [review]
LGTM

File: script/DeployLocal.s.sol [hunk 1]
```diff
@@ -7,6 +7,8 @@ import {DeployAaveV2FlashLoanCallback} from './callbacks/DeployAaveV2FlashLoanCa
 import {DeployAaveV3FlashLoanCallback} from './callbacks/DeployAaveV3FlashLoanCallback.s.sol';
 import {DeployBalancerV2FlashLoanCallback} from './callbacks/DeployBalancerV2FlashLoanCallback.s.sol';
 import {DeployRadiantV2FlashLoanCallback} from './callbacks/DeployRadiantV2FlashLoanCallback.s.sol';
+import {DeploySparkFlashLoanCallback} from './callbacks/DeploySparkFlashLoanCallback.s.sol';
+
```
Comment (Jeff-CCH) [inline script/DeployLocal.s.sol:11]: could be removed

File: script/DeployGnosis.s.sol [hunk 2]
```diff
@@ -0,0 +1,69 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+import {DeployCREATE3Factory} from './DeployCREATE3Factory.s.sol';
+import {DeployRouter} from './DeployRouter.s.sol';
+import {DeployAaveV3FlashLoanCallback} from './callbacks/DeployAaveV3FlashLoanCallback.s.sol';
+import {DeployBalancerV2FlashLoanCallback} from './callbacks/DeployBalancerV2FlashLoanCallback.s.sol';
+import {DeploySparkFlashLoanCallback} from './callbacks/DeploySparkFlashLoanCallback.s.sol';
+
+contract DeployEthereum is
```
[the remark this answers is not in the document]
Comment (zodahu) [reply script/DeployGnosis.s.sol]: nice catch

File: script/DeployLocal.s.sol (hunk 1 above)
Comment (zodahu) [reply script/DeployLocal.s.sol]: fixed

**Review by Jeff-CCH — APPROVED** [review]
LGTM

**Pull request merged**